packages/collector/test/tracing/logger/log4js/app.js

```diff
@@ -57,7 +58,11 @@ app.post('/log', (req, res) => {
     method = logger[level];
   }
 
-  args.push(message);
+  if (multipleArguments) {
+    args.push(message, 'more', 'arguments');
```

Comment:
This was enough to reproduce the `"TypeError: CreateListFromArrayLike called on non-object"` that has been mentioned in the original ticket.